File: AccuPay/Domain/TimeLogs/ITimeAttendanceHelper.vb
Imports AccuPay.Entity
Imports AccuPay.Helper.TimeLogsReader

Public Interface ITimeAttendanceHelper

    Function GenerateTimeLogs() As List(Of TimeLog)

    Function GenerateTimeAttendanceLogs() As List(Of TimeAttendanceLog)

    Function Analyze() As List(Of ImportTimeAttendanceLog)

End Interface

File: AccuPay.Core/Interfaces/ITimeAttendanceHelper.cs
using AccuPay.Core.Entities;
using AccuPay.Core.Services.Imports;
using System.Collections.Generic;

namespace AccuPay.Core.Interfaces
{
    public interface ITimeAttendanceHelper
    {
        List<TimeLog> GenerateTimeLogs();

        List<TimeAttendanceLog> GenerateTimeAttendanceLogs();

        List<TimeLogImportModel> Analyze();

        List<TimeLogImportModel> Validate();
    }
}
